configs/analysis_configs/feature_select_configs.yaml

```diff
@@ -0,0 +1,17 @@
+feature_select_configs:
+  params:
+    features: infer
+    image_features: False
+    samples: all
+    operation: variance_threshold
```

Comment:
There are multiple feature selection operations. How would it work if I wanted to use, say `variance_threshold` and `correlation_threshold`?

Reply:
Can you make the default to use all of the ones we used in the cell health project?

Reply:
after seeing how this file is used below, I think something like this would work:

```python
operation:
  - variance_threshold
  - correlation_threshold
  - blocklist
  - ...
```